test/interactions/hoverInteractionTests.ts

```diff
@@ -148,7 +148,7 @@ describe("Interactions", () => {
       assert.deepEqual(currentlyHovered.data, ["left", "right"],
                        "retrieves data corresponding to the current position");
 
-      triggerFakeMouseEvent("mouseout", hitbox, 400, 200);
+      triggerFakeMouseEvent("mouseout", hitbox, 401, 200);
```

Comment:
because otherwise the `mouseout` event occurs inside the hitbox, failing to actually trigger a `mouseout` with the new `Dispatcher`.